tests/PregTest.php

```diff
@@ -139,7 +139,7 @@ public function testPatternsValidation(string $pattern, ?int $expected = null, ?
         }
 
         if (!$setup()) {
-            $this->addToAssertionCount(1);
+            $this->expectNotToPerformAssertions();
```

Comment:
in both cases the test do assert what it is being tested no?

Reply:
it seems whole `if (!$setup()) {` is not needed